README.md

```diff
@@ -23,10 +23,11 @@ Machine: 2.8GHz AMD EPYC 7402P<br/>
 Configuration: Node v14.4, HTTP/1.1 without TLS, 100 connections, Linux 5.4.12-1-lts
 
 ```
-http - keepalive x 5,521 ops/sec ±3.37% (73 runs sampled)
-undici - pipeline x 9,292 ops/sec ±4.28% (79 runs sampled)
-undici - request x 11,949 ops/sec ±0.99% (85 runs sampled)
-undici - stream x 12,223 ops/sec ±0.76% (85 runs sampled)
+http - keepalive x 5,595 ops/sec ±3.33% (69 runs sampled)
+undici - pipeline x 7,055 ops/sec ±3.54% (79 runs sampled)
+undici - request x 11,551 ops/sec ±1.67% (81 runs sampled)
```

Comment:
It seems request is also regressing a bit?

Reply:
I will dig into this.

Reply:
any updates?

Reply:
not yet, benchmarks has a lot of variance between runs so it's a bit tricky. Still working on it.

Reply:
I think this is fixed sufficiently now.